src/components/ActionModal/ActionModal.tsx

```diff
@@ -115,7 +115,10 @@ export const ActionModal = ({
     >
       {serviceContainer ? (
         <div className="m-r-0">
-          <ServiceContainerCreatingUpdating {...serviceContainer} title={actionTitle}>
+          <ServiceContainerCreatingUpdating
+            {...{ ...serviceContainer, error: wereSubmitDataChanged ? '' : serviceContainer.error }}
```

Comment:
I would prefer more flat and easy read structure like `<ServiceContainerCreatingUpdating {...serviceContainer} error={...} title={actionTitle}>`, the same structure is already used also on other similar places, but this one is also possible.

Reply:
That's right, I just did not realize it.